projects/ids-enterprise-ng/src/lib/button/soho-button.component.ts

```diff
@@ -96,10 +96,22 @@ export class SohoButtonComponent implements AfterViewInit, OnDestroy, OnInit {
     this._buttonOptions.replaceText = replaceText;
     if (this.jQueryElement) {
       this.button.settings.replaceText = replaceText;
-      // todo: how to update the button replaceText changes?
+      // todo: how to update the button when replaceText changes?
```

Comment:
We probably should make this into a function https://github.com/infor-design/enterprise/blob/master/src/components/button/button.js#L67-L71 that can be called in the NG component

Reply:
Will raise another ticket for this ... on enterprise.